ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-701, 702] Always route to summary page from capital summary card actions

## Description of change
Always route to shopping basket summary page from the summary cards to ensure users don't go back to the select record type screen. This minimises the option persistence issues reported in qa. 

## Link to relevant ticket
[CRIMAPP-701](https://dsdmoj.atlassian.net/browse/CRIMAPP-701)
[CRIMAPP-702](https://dsdmoj.atlassian.net/browse/CRIMAPP-702)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/45662b13-7368-4f9c-a91b-863c9e19dd1f

## How to manually test the feature
Start a means tested application with a case type of either way, indictable or already in crown court (as these case types go through the full capital means assessment)
Enter a nino
Select no benefits
Fill in the information until you reach the capital screens (starting with assets)
Select no to assets, savings, national savings certificate and investments
When you get to the cya page, confirm that you can see the assets, savings, national savings certificate and investments sections
Check the change links take you to the shopping basket summary page and not the record type screen (e.g. 'which ___ does the client have' screen)
Then navigate to the review the application screen and confirm the same
Then go back and add an asset/saving/investment and then go to the cya or review screen and confirm that the edit link takes you to the shopping basket summary page

[CRIMAPP-701]: https://dsdmoj.atlassian.net/browse/CRIMAPP-701?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CRIMAPP-702]: https://dsdmoj.atlassian.net/browse/CRIMAPP-702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
